worker/host.go

```diff
@@ -33,12 +35,14 @@ type (
 	}
 
 	hostDownloadClient struct {
```

Comment:
Is there a particular reason you went for this solution rather than creating a separate `hostV2DownloadClient` and moving the `if hi.IsV2` check into `worker.Downloader`? 

I'm not sure that is a lot shorter in terms of loc but maybe slightly cleaner since we don't mix up different clients and price information within the same type.

Reply:
Splitting up the client into v1 and v2 cleaned it up a lot indeed. I don't see a good reason to move up the `IsV2` check though. The downloader doesn't have to be concerned about that, he only wants to download a sector. I moved it to `HostManager.Downloader`, which makes a lot of sense.